src/main/java/com/conveyal/gtfs/error/DuplicateTripError.java

```diff
@@ -12,21 +12,19 @@
     public static final long serialVersionUID = 1L;
 
     public final Priority priority = Priority.LOW;
-    public final String tripId;
     public final String duplicateTripId;
     public final String tripKey;
     public final String routeId;
 
-    public DuplicateTripError(String tripId, String duplicateTripId, String tripKey, String routeId) {
-        super("trips", 0, "trip_id");
-        this.tripId = tripId;
+    public DuplicateTripError(Trip trip, String duplicateTripId, String tripKey) {
+        super("trips", trip.sourceFileLine, "trip_id", trip.trip_id);
```

Comment:
Above comment about choosing which trip is the duplicate applies, but will have to be done where this is called.